docs/source/en/tei_cloud_run.md

```diff
@@ -0,0 +1,398 @@
+  - `--model-id`: The model ID to use, in this case, [`ibm-granite/granite-embedding-278m-multilingual`](https://huggingface.co/ibm-granite/granite-embedding-278m-multilingual).
+  - `--quantize`: The quantization method to use. If not specified, it will be retrieved from the `quantization_config->quant_method` in the `config.json` file.
+  - `--max-concurrent-requests`: The maximum amount of concurrent requests for this particular deployment. Having a low limit will refuse clients requests instead of having them wait for too long and is usually good to handle back pressure correctly. Set to 64, but default is 128.
+- `--port`: The port the container listens to.
+- `--cpu` and `--memory`: The number of CPUs and amount of memory to allocate to the container. Needs to be set to 4 and 16Gi (16 GiB), respectively; as that's the minimum requirement for using the GPU.
+- `--no-cpu-throttling`: Disables CPU throttling, which is required for using the GPU.
+- `--gpu` and `--gpu-type`: The number of GPUs and the GPU type to use. Needs to be set to 1 and `nvidia-l4`, respectively; as at the time of writing this tutorial, those are the only available options as Cloud Run on GPUs.
+- `--max-instances`: The maximum number of instances to run, set to 3, but default maximum value is 7. Alternatively, one could set it to 1 too, but that could eventually lead to downtime during infrastructure migrations, so anything above 1 is recommended.
+- `--concurrency`: the maximum number of concurrent requests per instance, set to 64. Note that this value is also aligned with the [`--max-concurrent-requests`](cli_arguments) argument in TEI.
+- `--region`: The region to deploy the Cloud Run service.
+- `--no-allow-unauthenticated`: Disables unauthenticated access to the service, which is a good practice as adds an authentication layer managed by Google Cloud IAM.
+
+> [!NOTE]
+> Optionally, you can include the arguments `--vpc-egress=all-traffic` and `--subnet=default`, as there is external traffic being sent to the public internet, so in order to speed the network, you need to route all traffic through the VPC network by setting those flags. Note that besides setting the flags, you need to set up Google Cloud NAT to reach the public internet, which is a paid product. Find more information in [Cloud Run Documentation - Networking best practices](https://cloud.google.com/run/docs/configuring/networking-best-practices).
+>
+> ```bash
+> gcloud compute routers create nat-router --network=default --region=$LOCATION
+> gcloud compute routers nats create vm-nat --router=nat-router --region=$LOCATION --auto-allocate-nat-external-ips --nat-all-subnet-ip-ranges
+> ```
+
+Finally, you can run the `gcloud run deploy` command to deploy TEI on Cloud Run as:
+
+```bash
+gcloud run deploy $SERVICE_NAME \
+    --image=$CONTAINER_URI \
+    --args="--model-id=$MODEL_ID,--max-concurrent-requests=64" \
+    --set-env-vars=HF_HUB_ENABLE_HF_TRANSFER=1 \
```

Comment:
Have you checked if this makes a difference ? I don't think TEI uses that flag at all (it downloads the models directly through Rust at load time)